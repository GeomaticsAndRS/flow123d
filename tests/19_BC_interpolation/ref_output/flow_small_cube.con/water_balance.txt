	"time"	"region"	"quantity"	"flux"	"flux_in"	"flux_out"	"mass"	"source"	"source_in"	"source_out"	"integrated_flux"	"integrated_source"	"error"
	0	"region_63"	"water_volume"	0	0	0	0	0	0	0	0	0	0
	0	"region_64"	"water_volume"	0	0	0	0	0	0	0	0	0	0
	0	"region_65"	"water_volume"	0	0	0	0	0	0	0	0	0	0
	0	"IMPLICIT BOUNDARY"	"water_volume"	3.51455e-11	-9.95217	9.95217	0	0	0	0	0	0	0
