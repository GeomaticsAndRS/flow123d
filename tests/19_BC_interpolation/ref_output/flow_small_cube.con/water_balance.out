# ---------------------------------------- WATER BALANCE ----------------------------------------
# Time of computed water balance: inf


# Boundary water balance:
# [boundary_id] [label]                  [total_balance][total_outflow][total_inflow][time]        
# -----------------------------------------------------------------------------------------------
  -3            IMPLICIT BOUNDARY          -8.01492e-10  9.95217       -9.95217      inf           
# -----------------------------------------------------------------------------------------------
# total boundary balance                   -8.01492e-10  9.95217       -9.95217      


# Source fluxes over material subdomains:
# [region_id]   [label]                  [total_balance]                            [time]        
# -----------------------------------------------------------------------------------------------
  63            region_63                  0                                         inf           
  64            region_64                  0                                         inf           
  65            region_65                  0                                         inf           
# -----------------------------------------------------------------------------------------------
# total sources balance                    0             


